cmd.sh

```diff
@@ -1,6 +1,44 @@
 #!/usr/bin/env sh
 
 set -e
+# first we check if a release already exists. We can't use the GET release by tag endpoint because
+# draft releases aren't tagged using the format we expect. Instead we list all releases and filter
+# using JQ to see if a release with the provided name already exists.
+listReleasesStatusCode=$(curl \
```

Comment:
We could consider breaking out into a github.release.get op and using from this op, but not a blocker

Reply:
IMO the philosophy of extremely small ops is pretty difficult to understand. I can split it up and get in line with the philosophy but I think it's nice that all of the behavior of this op is contained in a single repo. Dependency chains that require navigating to different repos to understand behavior is confusing and time consuming to track down